lib/src/rules/always_use_package_imports.dart

```diff
@@ -10,7 +10,9 @@ import '../ast.dart';
 
 const _desc = r'Avoid relative imports for files in `lib/`.';
 
-const _details = r'''*DO* avoid relative imports for files in `lib/`.
+const _details = r'''
+
```

Comment:
I actually don't understand why we need a _blank_ line after `r'''`. Can you explain?

Reply:
This was done, since most of the lints are doing so. I would suggest that we either remove all, or ensure all.

Reply:
I'm fine with whichever is the least amount of work (which is probably having the blank line, or you would have removed them to make it all consistent).

Reply:
No worry, replace by regular expression is fun to do :)

Reply:
Interesting...  I'll take a look at adding validation.

https://github.com/dart-lang/linter/issues/3709